crypto/property/property_parse.c

```diff
@@ -87,11 +87,13 @@ static int parse_name(OPENSSL_CTX *ctx, const char *t[], int create,
     int err = 0;
     size_t i = 0;
     const char *s = *t;
+    const char *start = *t;
```

Comment:
Here and throughout, you can just use `t` instead of storing it locally.
`t` is updated only after a successfully parsing something.

Reply:
That is not always true, for example here:

https://github.com/openssl/openssl/blob/master/crypto/property/property_parse.c#L113-L119

I decided to simply not risk it anywhere.  Using that name consistently everywhere makes for less confusing code.

Reply:
There were three cases of this and you've fixed two of them already :)

This parser is designed around checking what's coming but not updating the current position pointer until it's matched everything.

Reply:
Ok...  done